test/libsolidity/GasCosts.cpp

```diff
@@ -202,8 +203,9 @@ BOOST_AUTO_TEST_CASE(single_callvaluecheck)
 		}
 	)";
 	compileAndRun(sourceCode);
-	size_t bytecodeSizeNonpayable = m_compiler.object("Nonpayable").bytecode.size();
-	size_t bytecodeSizePayable = m_compiler.object("Payable").bytecode.size();
+	auto compilerOutput = m_compiler.output();
```

Comment:
```suggestion
	auto const& compilerOutput = m_compiler.output();
```

the way you did it will make a copy of the output